### PR TITLE
Update external-sharing-overview.md

### PR DESCRIPTION
**Pull request by StephenRiceMSFT**
Updates around how external sharing works when azure b2b integration is enabled or not.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointOnline/external-sharing-overview.md [hunk 1]
```diff
@@ -115,10 +115,11 @@ When users share files or folders, recipients will be asked to enter a verificat
 
 ![Enter Verification Code screen](media/verification-code.png)
 
-If the recipient has a work or school account, they only need to enter the code the first time. Then they will be added as a guest and can sign in with their organization's user name and password.
+If Azure B2B Integration is not enabled, recipients will enter the code each time they access the file or folder and are not added to your directory.
 
-If the recipient doesn't have a work or school account, they need to use a code each time they access the file or folder, and they are not added to your directory.
+If Azure B2B Integration is enabled, recipients will go through the Azure B2B invitation which may prompt them to sign-in or use a one time passcode, depending on your organization's settings in Azure B2B.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/external-sharing-overview.md:120]:
```suggestion
If Azure B2B Integration is enabled, recipients will go through the Azure B2B invitation which may prompt them to sign in or use a one-time passcode, depending on your organization's settings in Azure B2B.
```

File: SharePoint/SharePointOnline/external-sharing-overview.md [hunk 2]
```diff
@@ -115,10 +115,11 @@ When users share files or folders, recipients will be asked to enter a verificat
 
 ![Enter Verification Code screen](media/verification-code.png)
 
-If the recipient has a work or school account, they only need to enter the code the first time. Then they will be added as a guest and can sign in with their organization's user name and password.
+If Azure B2B Integration is not enabled, recipients will enter the code each time they access the file or folder and are not added to your directory.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/external-sharing-overview.md:118]:
```suggestion
If Azure B2B integration is not enabled, recipients will enter the code each time they access the file or folder and are not added to your directory.
```

File: SharePoint/SharePointOnline/external-sharing-overview.md (hunk 1 above)
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/external-sharing-overview.md:120]:
```suggestion
If Azure B2B integration is enabled, recipients will go through the Azure B2B invitation which may prompt them to sign in or use a one-time passcode, depending on your organization's settings in Azure B2B.
```

File: SharePoint/SharePointOnline/external-sharing-overview.md [hunk 3]
```diff
@@ -115,10 +115,11 @@ When users share files or folders, recipients will be asked to enter a verificat
...
 
-If the recipient doesn't have a work or school account, they need to use a code each time they access the file or folder, and they are not added to your directory.
+If Azure B2B Integration is enabled, recipients will go through the Azure B2B invitation which may prompt them to sign-in or use a one time passcode, depending on your organization's settings in Azure B2B.
 
+More information about this feature can be found at .[SharePoint and OneDrive integration with Azure AD B2B](./sharepoint-azureb2b-integration.md).)
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/external-sharing-overview.md:122]:
```suggestion
More information about this feature can be found at [SharePoint and OneDrive integration with Azure AD B2B](./sharepoint-azureb2b-integration.md).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @StephenRiceMSFT, kindly commit the minor suggested changes. Many thanks.

**Pull request merged**